.gitlab-ci-check-golang-unittests.yml

```diff
@@ -60,20 +60,20 @@ test:unit:
     # Get last remaining tags, if any.
     - git fetch --tags origin
 
-    # Prepare GOPATH for the build
+    # Prepare GOPATH
     - mkdir -p /go/src/github.com/mendersoftware
     - cp -r ${CI_PROJECT_DIR} /go/src/github.com/mendersoftware/${CI_PROJECT_NAME}
     - cd /go/src/github.com/mendersoftware/${CI_PROJECT_NAME}
   script:
-    - go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $? ;
+    - go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $?
```

Comment:
Question:
I know this was present previously, but why aren't we doing:
`go test -v -covermode=atomic -coverprofile=coverage.txt -coverpkg ./... ./...` or similar ?

Reply:
To be honest I don't know, this is inherited from Travis days. You suggestion might work well, do you care to launch an independent PR for that?

I only removed the trailing semi-colon that was annoying me.